lib/utils/getUnitFromValueNode.js

```diff
@@ -25,7 +25,7 @@ module.exports = function(node /*: Object*/) /*: ?string*/ {
 
   if (
     node.type !== "word" ||
-    !isStandardSyntaxValue(value) ||
+    !isStandardSyntaxValue(node.value) ||
```

Comment:
/cc @hudochenkov it is break other case, for example `.selector { property: 20px\9; }`

Reply:
But it's a hack for IE8.

All tests were passed.

Reply:
@hudochenkov yes, based, on code test should be failed :confounded: https://github.com/laysent/stylelint/blob/0c68261c43123df7e0ffaabbdc350a8a480fb43a/lib/utils/getUnitFromValueNode.js#L29, maybe postcss-value-parser recognize this as other type for node, i will looks on this today